test/core/security/alts_security_connector_test.cc

```diff
@@ -146,8 +125,8 @@ static void test_alts_peer_to_auth_context_success() {
   grpc_core::RefCountedPtr<grpc_auth_context> ctx =
       grpc_alts_auth_context_from_tsi_peer(&peer);
   GPR_ASSERT(ctx != nullptr);
-  GPR_ASSERT(test_identity(ctx.get(), TSI_ALTS_SERVICE_ACCOUNT_PEER_PROPERTY,
-                           "alice"));
+  GPR_ASSERT(test_identity_from_auth_context(
+      ctx.get(), TSI_ALTS_SERVICE_ACCOUNT_PEER_PROPERTY, "alice"));
```

Comment:
Is "alice" defined somewhere as a constant?

Reply:
I'll define "alice" and "unknown" as constants for the S2A tests (this particular file is an ALTS test).